pvlib/irradiance.py

```diff
@@ -351,7 +355,8 @@ def get_total_irradiance(surface_tilt, surface_azimuth,
         the list of accepted values.
     model : str, default 'isotropic'
         Irradiance model. Can be one of ``'isotropic'``, ``'klucher'``,
-        ``'haydavies'``, ``'reindl'``, ``'king'``, ``'perez'``.
+        ``'haydavies'``, ``'reindl'``, ``'king'``, ``'perez'``,
+        ``'perez-driesse'``.
```

Comment:
This might be the only instance of a hyphenated value. Pointing it out in case there's some reason, I don't know, to avoid this.

Reply:
Scipy has a variety of hyphenated method identifiers.